### PR TITLE
feat(integrations): add ability to log learning rate using WandbMetricsLogger

### PR DESCRIPTION
**Pull request by soumik12345**
Adds [GROWTH2-77](https://wandb.atlassian.net/browse/GROWTH2-77)

Description
-----------
Add support for tracking learning rate with [`WandbMetricsLogger `](https://github.com/wandb/wandb/blob/main/wandb/integration/keras/callbacks/metrics_logger.py)

Testing
-------
How was this PR tested?

Checklist
-------
- [x] Include reference to internal ticket "Fixes WB-NNNN" and/or GitHub issue "Fixes #NNNN" (if applicable)
- [x] Ensure PR title compliance with the [conventional commits standards](https://github.com/wandb/wandb/blob/master/CONTRIBUTING.md#conventional-commits)

cc: @ayulockin

### CODE REVIEW

**Comment by ayulockin** [comment]
Hey @soumik12345 LGTM. 

While you are at it, can you also expand the doc string to document the features of this callback? I seem to have missed fleshing out it in detail earlier.

You also need to write yea-test for it. You can modify [this](https://github.com/wandb/wandb/blob/main/tests/functional_tests/t0_main/keras/keras_metrics_logger_tf_2.6.yea) to include `:wandb:runs[0][summary][batch/learning_rate]: <some float>`.

File: wandb/integration/keras/callbacks/metrics_logger.py [hunk 1]
```diff
@@ -50,24 +50,34 @@ def __init__(
...
+    def _get_lr(self):
+        try:
+            return self.model.optimizer.learning_rate.numpy().item()
+        except AttributeError:
+            return self.model.optimizer.learning_rate(step=self.global_step).numpy()
+
     def on_epoch_end(self, epoch: int, logs: Optional[Dict[str, Any]] = None) -> None:
         """Called at the end of an epoch."""
         wandb.log({"epoch": epoch}, commit=False)
+        logs["learning_rate"] = self._get_lr()
```
Comment (ayulockin) [inline wandb/integration/keras/callbacks/metrics_logger.py:70]: put this inside an if-else block. When `isinstance(self.log_freq, int)`, we should not log the `learning_rate` here since it will be logged by `on_batch_end`

**Comment by ayulockin** [comment]
LGTM!

**Review by ayulockin — APPROVED** [review]

**Comment by ayulockin** [comment]
Hey @soumik12345, can you request @dmitryduev or @kptkin to review this PR. :)

**Comment by soumik12345** [comment]
Link to Sample Run: https://wandb.ai/geekyrakshit/keras-integration/runs/xn7c3p30

**Comment by ayulockin** [comment]
Hey @soumik12345, I was wondering why the Validate PR GitHub action failing and just realized that you have created this PR from your fork of the repo. You will have to close this PR and create a new PR from a branch within this (wandb) repo.

**Comment by soumik12345** [comment]
Re-raised PR at https://github.com/wandb/wandb/pull/4391
